prow/jenkins/controller_test.go

```diff
@@ -1050,3 +1052,65 @@ func TestOperatorConfig(t *testing.T) {
 		}
 	}
 }
+
+func TestSyncAbortedJob(t *testing.T) {
```

Comment:
Is it possible to test that a failure to abort the jenkins job on one run will lead to a re-try in the future?

Reply:
Not trivially in a unit test, because the logic for that is spread around:
* We call `syncAbortedJob` for aborted job that are not `completed` (the latter indicated by `.Status.CompletionTime` being non-nil
* in `syncAbortedJobs` we return errors and do not set the `.Status.CompletionTime` if we had one - For that I will add a test

Reply:
That's what I thought